# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/llvm/hotfix-abi.patch.powerpc64.elfv2
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- llvm-17.0.6.src/include/llvm/TargetParser/Triple.h.vanilla	2024-02-20 12:35:20.558296948 +0100
+++ llvm-17.0.6.src/include/llvm/TargetParser/Triple.h	2024-02-20 12:36:32.453290021 +0100
@@ -908,7 +908,7 @@
     return (getArch() == Triple::ppc64 &&
             ((getOS() == Triple::FreeBSD &&
               (getOSMajorVersion() >= 13 || getOSVersion().empty())) ||
-             getOS() == Triple::OpenBSD || isMusl()));
+             getOS() == Triple::OpenBSD || isMusl() || 1));
   }
 
   /// Tests whether the target 32-bit PowerPC uses Secure PLT.
